### PR TITLE
Feature/137 flagged services users page admin dashboard

### LINKED ISSUE #133: Admins should be able to delete services - frontend
Both admin dashboard & user detail

### PR DESCRIPTION
**Pull request by ZeynepDYilmaz**
this pr closes #133 and #137

### CODE REVIEW

File: backend/src/main/java/com/swe573/socialhub/controller/UserController.java [hunk 1]
```diff
@@ -65,8 +65,8 @@ public UserDto getUser(Principal principal, @PathVariable String userId) {
         }
     }
 
-    @DeleteMapping("/user/{userId}")
```
Comment (zynpgltkn) [inline backend/src/main/java/com/swe573/socialhub/controller/UserController.java:68]: I coded the admin dashboard according to @DeleteMapping code, why did we change it to PostMapping?

File: backend/src/main/java/com/swe573/socialhub/controller/ServiceController.java [hunk 2]
```diff
@@ -61,7 +61,7 @@ public ResponseEntity<ServiceDto> findServiceById(@PathVariable(value = "id") lo
         }
     }
 
-    @DeleteMapping("/{id}")
+    @PostMapping("/delete/{id}")
```
Comment (ecesari) [inline backend/src/main/java/com/swe573/socialhub/controller/ServiceController.java:64]: we've decided to use deletes instead of posts for backend, no?

Comment (sgbasaraner) [reply backend/src/main/java/com/swe573/socialhub/controller/ServiceController.java:64]: yes

Comment (zynpgltkn) [reply backend/src/main/java/com/swe573/socialhub/controller/ServiceController.java:64]: pinged deniz about this in case she didn't see here.

Comment (ZeynepDYilmaz) [reply backend/src/main/java/com/swe573/socialhub/controller/ServiceController.java]: sorry for the delay. fixed it.

**Pull request merged**